example.js

```diff
@@ -1,15 +1,29 @@
+const path = require('path')
 const screenshot = require('./')
-const fs = require('fs')
 
-screenshot().then((img) => {
-  fs.writeFile('out.jpg', img, function (err) {
-    if (err) {
-      throw err
+screenshot.listDisplays()
```

Comment:
Could this example be put in a separate file? The very basic example before is a great entrypoint. Maybe create an examples directory with the content there.